xml/ha_virtualization.xml

```diff
@@ -0,0 +1,484 @@
+        </para>
+      </step>
+      <step>
+        <para>
+          Open two terminals.
+        </para>
+      </step>
+      <step>
+        <para>
+          In the first terminal, connect to <literal>vm1</literal> via SSH.
+        </para>
+      </step>
+      <step>
+        <para>
+          In the second terminal, try to migrate <literal>vm1</literal> to node
+          <literal>&node2;</literal> with <command>crm resource move vm1 bob</command>.
+        </para>
+      </step>
+      <step>
+        <para>
+          Check the cluster status with <command>crm status</command>. Repeat this command
+          multiple times, until the cluster status stabilizes.
+        </para>
+      </step>
+      <step>
+        <para>
+          In the first terminal, check whether the SSH connection to <literal>vm1</literal>
+          is still active.
+        </para>
+      </step>
+      <step>
+        <para>
+          <emphasis role="bold">Expected result:</emphasis> The cluster status shows that
+          <literal>vm1</literal> has started on <literal>&node2;</literal>. The SSH connection
+          to <literal>vm1</literal> remains active during the whole migration.
+        </para>
+      </step>
+    </procedure>
+    <procedure>
+      <title>Verifying that the VM resource can move to another cluster node when the current node reboots</title>
+      <step>
+        <para>
+          The virtual machine <literal>vm1</literal> is running on node <literal>&node2;</literal>.
+       </para>
+      </step>
+      <step>
+        <para>
+          Reboot <literal>&node2;</literal>.
+        </para>
+      </step>
+      <step>
+        <para>
+          On node <literal>&node1;</literal>, check the cluster status with <command>crm status</command>.
+          Repeat this command multiple times, until the cluster status stabilizes.
+        </para>
+      </step>
+      <step>
+        <para>
+          <emphasis role="bold">Expected result:</emphasis> The cluster status shows that
+          <literal>vm1</literal> has started on <literal>&node1;</literal>.
+        </para>
+      </step>
+    </procedure>
+    <procedure>
+      <title>Verifying that the VM resource can move to another cluster node when the current node crashes</title>
+      <step>
+        <para>
+          The virtual machine <literal>vm1</literal> is running on node <literal>&node1;</literal>.
+        </para>
+      </step>
+      <step>
+        <para>
+          Simulate a crash on <literal>&node1;</literal> by forcing the machine off or
+          unplugging the power cable.
+        </para>
+      </step>
+      <step>
+        <para>
+          On node <literal>&node2;</literal>, check the cluster status with
+          <command>crm status</command>. Repeat this command multiple times, until the
+          cluster status stabilizes. Migrating the VM usually takes longer after a
+          node crashes than it does after a node reboots.
```

Comment:
Indeed, since technically it's now rather a "failover" than a "migrating" :-) FYI in detail, it involves detection and declaration of node loss, reforming of cluster membership, fencing of the lost node, and takeover of resources. But anyway we shouldn't need to give the detail here.

Reply:
Thanks, I'll rephrase :)